### PR TITLE
Network url in explorer (camino-block-explorer)

### PR DESCRIPTION
**Pull request by Esteban100k**
With this PR, both the URL problem and the bugs generated by this change are solved.
Now the urls will have the network that is currently in the explorer, for example /explorer/camino

It is necessary to review the PRs of the other suite projects related to this branch.

### CODE REVIEW

File: src/api/index.tsx [hunk 1]
```diff
@@ -123,10 +126,12 @@ export async function loadCAddressTransactions({
                             : `Failed-${parseInt(transaction.status)}`,
                     timestamp: parseInt(transaction.timestamp) * 1000,
                     to: transaction.to,
-                    value: parseInt(transaction.value),
-                    transactionCost: BigNumber(transaction.gasUsed)
-                        .multipliedBy(BigNumber(transaction.effectiveGasPrice))
-                        .toNumber(),
+                    value: getDisplayAmount(parseInt(transaction.value)).value,
```
Comment (aeddaqqa) [inline src/api/index.tsx]: why this needs to be changed ?

File: src/utils/route-utils.ts [hunk 2]
```diff
@@ -1,33 +1,40 @@
...
-    XADDRESS,
-    PADDRESS,
-    XTRANSACTION,
-    PTRANSACTION,
-    TRANSACTION,
-    BLOCK,
-    ADDRESS,
-    BASE_PATH,
-} from './route-paths'
+import { RoutesConfig } from './route-paths'
```
Comment (aeddaqqa) [inline src/utils/route-utils.ts]: remove unused

File: src/app/index.tsx [hunk 3]
```diff
@@ -12,49 +12,75 @@ import { TransactionDetails, BlockDetails } from './pages/CChainPages'
...
-                    <Route path={`${BASE_PATH}/mainnet`} element={<ComingSoonPage />} />
-                    <Route path={`${BASE_PATH}/validators`} element={<Validators />} />
-                    <Route path={`${BASE_PATH}/statistics`} element={<Statistics />} />
-                    <Route path="*" element={<NotFoundPage />}></Route>
-                </Route>
-            </Routes>
-            <GlobalStyle />
-        </BrowserRouter>
+        <>
+            {activeNetwork ? (
```
Comment (aeddaqqa) [inline src/app/index.tsx]: the helmet should not be wrapped with state

File: src/app/pages/CChainPages/CChainPage.tsx [hunk 4]
```diff
@@ -13,6 +13,14 @@ import { Status } from 'types'
 import LoadingWrapper from 'app/components/LoadingWrapper'
 
 export default function CChainPage() {
+    const [componentLoaded, setComponentLoaded] = React.useState(false)
```
Comment (aeddaqqa) [inline src/app/pages/CChainPages/CChainPage.tsx]: maybe setTimout needs to be refactored or search for another solution
* metaData is inside the state
* loading c-chain details is done after the chains is loaded
* the components on this page have their own loading
* setTimout can be a problem if the connexion of the user is bad (400ms is not enough)

**Review by aeddaqqa — CHANGES_REQUESTED** [review]

**Review by aeddaqqa — APPROVED** [review]

**Comment by mo-c4t** [comment]
always having the lower case of the network name will not work for example:
http://localhost:5001/Explorer/columbus/statistics  >> This works
http://localhost:5001/Explorer/COLUMBUS/statistics  >> This works
http://localhost:5001/Explorer/Columbus/statistics  >> This doesn't works

the url should redirect to lowercase

**Pull request merged**